src/admin/components/payments_table/index.js

```diff
@@ -35,10 +36,14 @@ var PaymentsTable = (
       'Reference', 'Notes', 'Delete']
 
   var entries = make_payments_with_balance(payments)
-      .slice(1)
-      .map((payment) =>
-        headers.map(get_entry_for_payment(payment, remove_payment)))
-
+  .slice(1)
+  .map((payment) => {
+    const convertedPayment = {...payment
```

Comment:
I'm not sure the change should live here. It makes more sense to me in `get_entry_for_payment`.